packages/components/src/components/combobox/component.tsx

```diff
@@ -0,0 +1,528 @@
+		const hasExpertSlot = showExpertSlot(this.state._label);
+		const { ariaDescribedBy } = getRenderStates(this.state);
+
+		return (
+			<Host class={{ 'kol-combobox': true, combobox: true }}>
+				<div class="combobox">
+					<KolInputTag
+						_accessKey={this.state._accessKey}
+						_disabled={this.state._disabled}
+						_hideError={this.state._hideError}
+						_hideLabel={this.state._hideLabel}
+						_hint={this.state._hint}
+						_icons={this.state._icons}
+						_id={this.state._id}
+						_label={this.state._label}
+						_msg={this.state._msg}
+						_required={this.state._required}
+						_tooltipAlign={this._tooltipAlign}
+						_touched={this.state._touched}
+						onClick={() => this.ref?.focus()}
+						role={`presentation` /* Avoid element being read as 'clickable' in NVDA */}
+					>
+						<span slot="label">
+							{hasExpertSlot ? (
+								<slot name="expert"></slot>
+							) : typeof this.state._accessKey === 'string' ? (
+								<>
+									<InternalUnderlinedAccessKey accessKey={this.state._accessKey} label={this.state._label} />{' '}
+									<span class="access-key-hint" aria-hidden="true">
+										{this.state._accessKey}
+									</span>
+								</>
+							) : (
+								<span>{this.state._label}</span>
+							)}
+						</span>
+						<div slot="input">
+							<div class="combobox__group">
+								<input
+									class="combobox__input"
+									type="text"
+									role="combobox"
+									aria-autocomplete="both"
+									aria-expanded="false"
+									aria-controls="listbox"
+									value={this.state._value}
+									title=""
+									accessKey={this.state._accessKey}
+									aria-describedby={ariaDescribedBy.length > 0 ? ariaDescribedBy.join(' ') : undefined}
+									aria-label={this.state._hideLabel && typeof this.state._label === 'string' ? this.state._label : undefined}
+									autoCapitalize="off"
+									autoCorrect="off"
+									disabled={this.state._disabled}
+									id={this.state._id}
+									name={this.state._name}
+									required={this.state._required}
+									spellcheck="false"
+									{...this.controller.onFacade}
+									onInput={this.onInput.bind(this)}
+									onClick={this.toggleListbox.bind(this)}
+									onChange={this.onChange.bind(this)}
+								/>
+								<span onClick={this.toggleListbox.bind(this)} tabIndex={0} class="combobox__icon">
+									<KolIconTag _icons="codicon codicon-triangle-down" _label={`dropdown`} />
+								</span>
+							</div>
+							<ul role="listbox" aria-label="" class={{ combobox__listbox: true, 'combobox__listbox--hidden': !this._isOpen }} tabindex="-1">
+								{this._filtredOptions &&
+									this._filtredOptions.map((option, index) => {
+										const key = `-${index}`;
+										return (
+											<li key={key} data-index={index} tabIndex={0} onClick={() => this.selectOption(option)} class="combobox__item">
+												{option}
+											</li>
+										);
+									})}
```

Comment:
Variablen bitte nur verwenden, falls der Wert wiederverwendet wird.
```suggestion
{this._filtredOptions && this._filtredOptions.map((option, index) => (
									<li key={`-${index}`} data-index={index} tabIndex={0} onClick={() => this.selectOption(option)} class="combobox__item">
										{option}
									</li>
								))}
```